src/Microsoft.DotNet.Darc/src/DarcLib/AzureDevOpsClient.cs

```diff
@@ -286,13 +288,41 @@ public async Task<string> CreatePullRequestAsync(string repoUri, PullRequest pul
             VssConnection connection = CreateVssConnection(accountName);
             GitHttpClient client = await connection.GetClientAsync<GitHttpClient>();
 
+            // get the identity client used to resolve user name strings to identities
+            var identityClient = await connection.GetClientAsync<IdentityHttpClient>();
+
+            // Get the identities of each reviewer
+            var identities = new List<IdentityRefWithVote>();
+            foreach (var reviewer in pullRequest.Reviewers)
+            {
+                // look for a matching display name
+                var matchingIdentity = (await identityClient.ReadIdentitiesAsync(IdentitySearchFilter.DisplayName, reviewer).ConfigureAwait(false)).FirstOrDefault();
```

Comment:
>.ConfigureAwait(false) [](start = 124, length = 22)

Why are their configure awaits only on these methods> Three are other awaits in this method... in fact, there are other awaits this change added.